templates/config.json.j2

```diff
@@ -1,25 +1,27 @@
 {
-    "masters": [{% for host in groups['mesosmaster'] %}"{{hostvars[host]['ansible_default_ipv4']['address']}}:5050"{% if not loop.last %},{% endif %}{% endfor %}],
```

Comment:
> what are the pros of having this as a variable rather than to deduce it from facts

you are right it is nice that mesosmaster's are taken from hosts but on the other hand it adds dependency, so all playbook users will have to have that.